app/javascript/src/Form/FormFieldset.jsx

```diff
@@ -0,0 +1,68 @@
+// @flow
+// TODO: Replace this component when patternfly-react implements it.
```

Comment:
Nope, not a single thing. Also, I didn't open one because they don't have it in their library pattern for v4 nor v3 (following they contribution guidelines). I've asked about this in their slack, but no answer.